ng-dialog/ng-dialog.d.ts

```diff
@@ -7,10 +7,21 @@
 
 declare module angular.dialog {
 
+	/*
+	 * Everytime ngDialog is opened or closed we're broadcasting three events
+	 * (dispatching events downwards to all child scopes):
+	 *
+	 * for more info see: https://github.com/likeastore/ngDialog#events
+	 */
+	export const EVENT_OPENEND:string = 'ngDialog.opened';
```

Comment:
I don't think you are able to do any constants (code) within the d.ts file, only a contract (definition).